source/_integrations/recorder.markdown

```diff
@@ -63,6 +63,11 @@ recorder:
       required: false
       default: true
       type: boolean
+    auto_repack:
+      description: Automatically repack the database every second sunday after the auto purge. Without a repack, the database may not decrease in size even after purging, which takes up disk space and can make Home Assistant slow. If you disable `auto_repack` it is recommended that you create an automation to call the [`recorder.purge`](#service-purge) periodically. This flag has no effect if `auto_purge` is disabled.
```

Comment:
```suggestion
      description: Automatically repack the database every second sunday after the auto purge. Without a repack, the database may not decrease in size even after purging, which takes up disk space and can make Home Assistant slow. If you disable `auto_repack` it is recommended that you create an automation to call the [`recorder.purge`](#service-purge) periodically. This flag has no effect if `auto_purge` is disabled or the database engine is not sqlite.
```

Reply:
https://github.com/home-assistant/core/pull/69331

Reply:
Resolving this. Will open a new PR if we decide we need to move forward with https://github.com/home-assistant/core/pull/69331